plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -121,6 +121,7 @@
 import static com.google.common.collect.ImmutableSet.toImmutableSet;
 import static io.trino.plugin.hive.HiveApplyProjectionUtil.extractSupportedProjectedColumns;
 import static io.trino.plugin.hive.HiveApplyProjectionUtil.replaceWithNewVariables;
+import static io.trino.plugin.hive.HiveMetadata.TRINO_QUERY_ID_NAME;
```

Comment:
The value of the query id constant will not land in Hive Metastore. Please consider moving it to trino-iceberg.

Reply:
I moved the constant into the class that it's used in (this one), so that it would be in the trino-iceberg project only.

However, I'm not sure this is really the best place for it. It looks kind of lonely. Let me know if I should place it somewhere else or possibly make a new class for iceberg snapshot summary properties.